doc/algorithms/index.rst

```diff
@@ -1,20 +1,15 @@
 ######################
-First-Party Algorithms
+Built-in Algorithms
 ######################
```

Comment:
I'm pretty sure you need the `#`'s to be the same length as the text. That is probably one of the reasons why your codebuild is failing.

Reply:
I think it needs to be the same or longer in length. `make html` as well as `tox -e twine,sphinx` pass locally. The build errors seem unrelated to my changes.

Reply:
sounds good to me